retis/src/core/events/bpf/include/events.h

```diff
@@ -7,17 +7,23 @@
 
 /* Please keep the below in sync with its Rust counterpart. */
 #define EVENTS_MAX		8 * 1024
-#define RAW_EVENT_DATA_SIZE	1024 - 2 /* Remove the size field */
+#define RAW_EVENT_DATA_SIZE	1024 - 8 /* Remove the size and pad fields */
```

Comment:
We could introduce an `offsetof` macro and use it here for better maintenance?

Reply:
not sure `offsetof` can be used here as we need to define this macro getting the offset of a member using the macro in the definition :)

Reply:
a quick check confirm this is a problem for the reason I explained above

Reply:
Thanks for checking!